docs/install/install-from-packages.md

```diff
@@ -145,20 +145,21 @@ npm install -g yo
 
 (install-prerequisites-yarn-label)=
 
-#### Yarn
+#### Yarn 3
 
-Install the Yarn Classic version (not the latest 2.x one) using `npm`.
+Install the latest Yarn 3 version (not the Classic 1.x one) using `npm`.
 
 1.  Open a terminal and type:
 
     ```shell
-    npm install yarn@1
+    npm install yarn@3
     ```
 
-2.  Verify that Yarn v1.x.x is installed and activated.
+2.  Verify that Yarn v3.x.x is installed and activated.
 
     ```shell
     yarn -v
+    3.2.3
```

Comment:
Commands go in `shell`, output to `console`.

````suggestion
    ```
    ```console
    3.2.3
````

Reply:
In the other console outputs, both commands and outputs are copy/pasted in one block, for example the whole cookiecutter flow. That's the example I followed.

Reply:
Which ones? If syntaxes are mixed in a block, then they need to be broken apart.